types/chrome/test/index.ts

```diff
@@ -1853,6 +1853,16 @@ async function testBrowsingDataForPromise() {
     await chrome.browsingData.removeIndexedDB({});
 }
 
+// https://developer.chrome.com/docs/extensions/reference/cookies
+async function testCoookies() {
```

Comment:
Maybe merge `testCoookies` into `testCookieForPromise` :)

Reply:
🤦‍♂️
I swear I have looked for existing cookies tests in this file

Reply:
done